common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/Image.scala

```diff
@@ -30,24 +30,29 @@ case class MergedImage[Id <: IdState](
       id = id,
       location = location
     )
+}
 
-  def augment(inferredData: => Option[InferredData]): AugmentedImage[Id] =
-    AugmentedImage[Id](
-      id = id,
-      location = location,
-      parentWork = parentWork,
-      fullText = fullText,
-      inferredData = inferredData
-    )
+object MergedImage {
+  implicit class IdentifiedMergedImageOps(
+    mergedImage: MergedImage[Identified]) {
+    def augment(inferredData: => Option[InferredData]): AugmentedImage =
+      AugmentedImage(
+        id = mergedImage.id,
+        location = mergedImage.location,
+        parentWork = mergedImage.parentWork,
+        fullText = mergedImage.fullText,
+        inferredData = inferredData
+      )
+  }
```

Comment:
Why is it necessary to make this an implicit class?

Reply:
It's not necessary strictly speaking. It is necessary to be able to do `image.augment` which is already used in the inferer_manager. Previously you could simply have `augment` as a method of `MergedImage` but now that's not possible because `AugmentedImage` is always `Identified` while `MergedImage` can be `Identifiable` or `Identified` so you can only call `augment` on a `MergedImage[Identified]` but not on a `MergedImage[Identifiable]`. The implicit class allows that while retaining the syntax `image.augment`